string name
pcl_msgs/PolygonMesh mesh
geometry_msgs/Twist displacement